benefits/static/css/admin/styles.css

```diff
@@ -86,20 +86,12 @@ input[type="password"]:focus,
 }
 
 /* Eligibility Page */
-.checkbox-parent .form-group:last-of-type .col-12 {
+/* Checkbox form fields should show the checkbox first, then label */
+.form-field-container .form-group:has(input[type="checkbox"]) {
```

Comment:
The current selector is selecting for the last `.form-group` div in the container. This code will break if we ever add more form fields below the checkbox. I wanted to write CSS that didn't require adding a new class (like `checkbox-parent`), and also, did not require knowing that the checkbox field is last. So I came up with this:

"Look for all `.form-group` classes within the `.form-field-container` parent, that *has* an `input` that is a `checkbox`."

Learn how to use the `has` selector here: https://css-tricks.com/the-css-has-selector/